docs/specs/git.yml

```diff
@@ -313,6 +314,25 @@ outputs:
       "gitcommit": "https://github.com/testowner/testrepo/blob/e662dbc7c5e582a14b863d6a6a82e495d6272f91/docs/index.md",
     }
 ---
+# Specify editLink repo and branch in .openpublishing.publish.config.json
+repos:
+  https://github.com/testowner/test-op-edit-link-repo#master-edit-link:
+  - files:
+      .openpublishing.publish.config.json: |
+        {
+          "git_repository_url_open_to_public_contributors": "https://github.com/a/b",
```

Comment:
there is a `open_to_public_contributors` under `docsets_to_publish`, what is the relationship with these two here?
https://github.com/OPS-E2E-PPE/edge-developer/blob/f05725de0970b9bc6c5405f8228d2f6c291d648d/.openpublishing.publish.config.json#L8

Reply:
It is a shortcut for `globalMetadata.open_to_public_contributors` https://github.com/dotnet/docfx/blob/v3/src/docfx/config/ops/OpsConfig.cs#L70